requirements.txt

```diff
@@ -3,7 +3,7 @@ numpy==1.21.6;python_version<"3.10"
 numpy;python_version=="3.10"
 cython==0.29.28
 scipy>=1.4.0
-torch>=1.7
+torch==1.10.0
```

Comment:
Let's go with 1.13.1

Reply:
Done. I tested the models with 1.13.1 and they work fine. I updated the requirements and install commands.